### ISSUE TITLE
Add dc.identifier.orcid to the default submission form

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#385]
Add to alphabetical list in  `vtechworks/dspace/config/input-forms.xml`

```
<form-value-pairs>
        <value-pairs value-pairs-name="common_identifiers"
dc-term="identifier">
```

displayed value="ORCID"

**Comment by alawvt** [comment]
Also, in the same file and form-value-pairs section, list "Report" before "Software" in content_types.

**Comment by AnushaPillai** [comment]
Resolved with Pull request #394
